tests/external_board_dirs/netdev-ci-boards/nrf52840-ci-eth/Makefile.features

```diff
@@ -0,0 +1,8 @@
+CPU_MODEL = nrf52840xxaa
```

Comment:
Since the static tests fail and this is just a copy of an existing board maybe we do what was done on the esp:
```suggestion
include $(RIOTBOARD)/nrf52840dk/Makefile.features
```